src/components/Canvas/FallingBall.jsx

```diff
@@ -0,0 +1,58 @@
+// @ts-check
+
+import { CanvasBuilder } from './CanvasBuilder'
+import { GRAVITY, BLUE } from './Standards'
```

Comment:
I love the idea of standards or constants.

But I think we should have multiple files or exports of object enums. Like

```js
export const Physics = Object.freeze({
  GRAVITY: { x: 0, y: 0.1 }
})

export const Color = Object.freeze({
  // ...
})
```

Reply:
And we should think carefully about what these constants "being" to. Like, should that be maintained in this `/components` folder? Or should they be more top-level?

Reply:
OK - I need more education on these object enums.
And, yes, let's discuss where to hold things like this as well.

Reply:
It's just a structural / design choice. We don't technically need `Object.freeze()` but it ensures we never reassign these object properties.

Something like:

```js
export const Color = Object.freeze({
  RED: 'rgb(212, 42, 32)',
  YELLOW: 'rgb(250, 194, 43)',
  BLUE: 'rgb(14, 99, 142)'
})
```

Is manifest in code like:

```js
import { Physics, Color } from './Standards'

const ball = {
    color: Color.BLUE,
    // ...
    update () {
      this.velocity.x += Physics.GRAVITY.x
      this.velocity.y += Physics.GRAVITY.y
      // ...
```

The enclosing objects provide a "namespace" for our enumerated constants.

Reply:
Closing this thread. I added an issue to iterate on this later

https://github.com/peterjmartinson/huygens-site/issues/14